modules/database/src/ioc/rsrv/camessage.c

```diff
@@ -791,9 +791,11 @@ static int write_action ( caHdrLargeArray *mp,
         return RSRV_ERROR;
     }
 
-    asWritePvt = asTrapWriteWithData ( pciu->asClientPVT,
+    asWritePvt = asTrapWriteWithDataX ( pciu->asClientPVT,
```

Comment:
CA does not, and never will, have a concept of RPC call.  and imo. never will support TLS transport.  So I think the changes under rsrv/ should be removed as proof of API backwards compatibility.